src/js/components/FormField/__tests__/__snapshots__/FormField-test.js.snap

```diff
@@ -558,6 +558,7 @@ exports[`FormField disabled 1`] = `
 >
   <div
     className="c1 "
+    disabled={true}
```

Comment:
I don't think we should add the HTML disabled attribute to the DOM, the DOM might override the grommet styling for the disabled state which I'm not sure we should allow. 
Was there a reason this was added?  I might be missing something.

Reply:
You are right. This was unnecessary and has been removed.